modules/ovirt-csi-driver-storage-class.adoc

```diff
@@ -1,30 +1,42 @@
 // Module included in the following assemblies:
 //
 // * storage/container_storage_interface/persistent-storage-csi-ovirt.adoc
+// * post_installation_configuration/storage-configuration.adoc
 
 [id="ovirt-csi-driver-storage-class_{context}"]
-= oVirt CSI driver storage class
+ifeval::["{context}" == "post-install-storage-configuration"]
+= {rh-virtualization-first} object definition
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-ovirt"]
+= {rh-virtualization-first} CSI driver storage class
+endif::[]
+
 
 {product-title} creates a default object of type `StorageClass` named `ovirt-csi-sc` which is used for creating dynamically provisioned persistent volumes.
 
 To create additional storage classes for different configurations, create and save a file with the `StorageClass` object described by the following sample YAML:
 
+.ovirt-storageclass.yaml
 [source,yaml]
 ----
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: <storage-class-name>  <1>
   annotations:
-    storageclass.kubernetes.io/is-default-class: "false"  <2>
+    storageclass.kubernetes.io/is-default-class: "<boolean>"  <2>
 provisioner: csi.ovirt.org
+allowVolumeExpansion: <boolean> <3>
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
 parameters:
-  storageDomainName: <rhv-storage-domain-name> <3>
-  thinProvisioning: "true"  <4>
-  csi.storage.k8s.io/fstype: ext4 <5>
+  storageDomainName: <rhv-storage-domain-name> <4>
+  thinProvisioning: "<boolean>"  <5>
+  csi.storage.k8s.io/fstype: <file_system_type> <6>
 ----
 <1> Name of the storage class.
 <2> Set to `false` if the storage class is the default storage class in the cluster. If set to `true`, the existing default storage class must be edited and set to `false`.
-<3> {rh-virtualization} storage domain name to use.
-<4> Disk must be thin provisioned.
-<5> File system type to be created.
+<3> `true` enables dynamic volume expansion, `false` prevents it. `true` is recommended.
+<4> The {rh-virtualization} storage domain name to use.
+<5> If `true`, the disk is thin provisioned. If `false`, the disk is preallocated. Thin provisioning is recommended.
+<6> File system type to be created. Possible values: `ext4` or `xfs`.
```

Comment:
this parameter is optional, default is ext4 (user don't have to add this row to yaml)